process/graduation_criteria.md

```diff
@@ -24,8 +25,8 @@ See the [CNCF Sandbox Guidelines v1.0](https://github.com/cncf/toc/blob/main/pro
 To be accepted to incubating stage, a project must meet the sandbox stage requirements plus:
 
  * Document that it is being used successfully in production by at least three independent direct adopters which, in the TOC’s judgement, are of adequate quality and scope. For the definition of an adopter, see https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter.
-
  * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+ * Explicitly define the criteria, process and offboarding or emeritus conditions for project committers/maintainers; or those who may interact with the CNCF on behalf of the project. The list of maintainers should be preferably be stored in a MAINTAINERS.md file and audited at a minimum of an annual cadence.
```

Comment:
Is there a good reason not to upgrade the existence and review of MAINTAINERS.md from "should" to "must"?

Reply:
Please don't apply "must" to the file format. Istio keeps its list in yaml format as it is an input to [peribolos](
https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/).